frontend/main/components/ProfilePage/ProfilePage.css

```diff
@@ -0,0 +1,156 @@
+/*.header {
+  background-color: #232C3C;
+  color: white;
+  padding: 15px 20px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  font-size: 24px;
+  font-weight: bold;
+}
+
+.nav a {
+  margin: 0 15px;
+  color: white;
+  text-decoration: none;
+  font-size: 16px;
+}
+
+.nav a:hover {
+  text-decoration: underline;
+}*/
+.public-info {
+    background-color: #5B91C8;
+    padding: 70px 100px;
+    text-align: center;
+    position: relative;
+    display: flex;
+    justify-content: center; 
+  }
+  
+  .profile-avatar {
+    width: 120px;
+    height: 120px;
+    border-radius: 50%;
+    margin-bottom: 20px;
+  }
+  
+.info-container {
+    display: flex;
+    align-items: center; 
+    gap: 20px;
+}
+
+.profile-avatar {
+    width: 100px;
+    height: 100px; 
+    border-radius: 50%;
+    object-fit: cover; 
+    border: 2px solid #232C3C;
+    position: absolute;
+    top: 17px;
+    left: 40px;
+}
+
+.user-details h1 {
+    margin: 0;
+    font-size: 24px;
+    position: fixed;
+    top: 20px;
+    left: 180px;
+  }
+  
+.user-details p:nth-child(2) {
+    position: absolute; 
+    top: 60px;
+    left: 180px;
+    margin: 0;
+    font-size: 16px;
+    color: #FFFFFF;
+}
+.user-details p:nth-child(3) {
+  position: absolute;
+  top: 80px;
+  left: 180px;
+  margin: 0;
+  font-size: 16px; 
+  color: #FFFFFF;
+}
+  
+  .edit-icon {
+    cursor: pointer;
+    font-size: 14px;
+    margin-left: 10px;
+    color: #007bff;
+    position: fixed;
+    top: 23px;
+    left: 350px;
+  }
+  
+  
+  
+  .private-sections .section {
+    background: white;
+    margin: 20px auto;
+    padding: 20px;
+    border: 1px solid #ddd;
+    border-radius: 5px;
+    width: 80%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+  
+  .section h2 {
+    margin: 0;
+    font-size: 18px;
+    font-weight: bold;
+  }
+  
+  .section .edit-icon {
+    font-size: 18px;
+    color: #007bff;
+  }
+  
+  .section .edit-icon:hover {
+    color: #0056b3;
+    cursor: pointer;
+  }
+  
+  .order-history {
+    margin: 20px auto;
+    padding: 10px;
+    width: 80%;
+  }
+  
+  .order-history h2 {
+    font-size: 20px;
+    margin-bottom: 20px;
+    text-align: center;
+  }
+  
+  .order-list {
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    gap: 20px;
+  }
+  
+  .order-item {
```

Comment:
Currently the style for the order items looks very strange. Can you fix the CSS so the text no longer spills out of the box?